### PR TITLE
Kevingranade balance draft

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Purpose of change
Added a draft balance document to outline intended game balance.

### CODE REVIEW

**Comment by actual-nh** [comment]
Why not incorporate into GAME_BALANCE.md?

File: balance.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Improvised melee weapons such as sticks or primitive spears greatly improve on unarmed, to the point where unarmed can be considered a dead end.  
+Improvised weapons suffer from relative fragility, and should be replaced with a purpose built melee weapon as soon as possible.  
+Repurposed tools such as hammers and machetes, as well as survivor-crafted equivalents represent an intermediate tier of melee weapon that can be relied on long-term, but are still outclassed by purpose-built melee weapons.  
+At the pinnnacle of melee weapons are purpose-built melee weapons such as swords, robust spears, polearms and axes.  These weapons will reliably outperform all kinds of repurposed weapons, but will have some differentiation in how effective they are based on the situation.  
+Martial arts training and skill can extend the usefullness of melee combat of all kinds, but not to the point where melee combat can become the player's sole form of combat.  
+The intended niche for melee combat of all kinds is to conserve other resources when dealing with relatively low-threat opponents, or as a defense when caught unprepared.
```
Comment (actual-nh) [inline balance.md:22]: Enemies that are difficult (or have side effects) to use guns against - possibility?

**Comment by kevingranade** [comment]
> Why not incorporate into GAME_BALANCE.md?

Because that's mostly aimed at contributors, and this is more explaining the rationale to a more general audience.

File: balance.md (hunk 1 above)
Comment (I-am-Erk) [reply balance.md:22]: Also close quarters fighting when you need to be silent, for example in a lab.

**Comment by I-am-Erk** [comment]
I'd suggest making sure to emphasize in each section that no tool (looking at guns here) should be even close to the "right" answer: guns should be loud, ammo should have some degree of rarity, and the scenario and enemy design should emphasize establishing situations where guns aren't the obvious first choice.  That should be a throughline everywhere, you should never be able to set up a single loadout that solves every problem.

**Pull request closed without merging**